cmc-domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/offers/Settlement.java

```diff
@@ -33,11 +34,27 @@ public void reject(MadeBy party) {
     @JsonIgnore
     PartyStatement getLastStatement() {
         if (partyStatements.isEmpty()) {
-            throw new IllegalSettlementStatementException("No statements have yet been made during that settlement");
+            throw new IllegalSettlementStatementException(NO_STATEMENTS_MADE);
         }
         return partyStatements.get(partyStatements.size() - 1);
     }
 
+    @JsonIgnore
+    public PartyStatement getLastStatementWithOffer() {
```

Comment:
not sure whether to do this or just do:
```
return partyStatements.get(partyStatements.size() - 2);
```

Reply:
I think this is fine. I'd rename the method to just `getLastOffer` though.

Reply:
Or `getLastOfferStatement`.

Reply:
done the second one